strcase/id.go

```diff
@@ -0,0 +1,120 @@
+package strcase
+
+import (
+	"math"
+	"strings"
+	"unicode"
+)
+
+func ToPascalCase(input string) string {
```

Comment:
I think something should say that those functions respect the Go initialisms pattern. 
Either the package name, the method name, or at the very least the doc

Reply:
I could see something wanting or not wanting the go initialisms. 
How about:
- `ToPascalCase`
- `ToPascalGoCase`
- `ToCamelCase`
- `ToCamelGoCase`
- `ToSnakeCase`

Reply:
👍